dependency_check_suppressions.xml

```diff
@@ -91,4 +91,14 @@
     <cpe>cpe:/a:processing:processing</cpe>
     <vulnerabilityName>CVE-2018-1000840</vulnerabilityName>
   </suppress>
+
+  <!-- Dokka -->
+  <suppress>
```

Comment:
Please add the end date, and there's already an existing Dokka block, please append there. Thank you

Reply:
updated